### PR TITLE
fix(core): update styles

### PR DESCRIPTION
**Pull request by eckdanny-osi**

### CODE REVIEW

File: libs/components/src/Layouts/Layout.module.scss [hunk 1]
```diff
@@ -42,15 +42,15 @@ $offset: $banner-height + map-get($spacers, 3);
   }
 
   .PageHeaderContainer {
-    @include gradient-x($info, $primary);
+    background-color: $banner-page-header-bg;
```
Comment (eckdanny-osi) [inline libs/components/src/Layouts/Layout.module.scss]: no... undo this.

File: libs/components/src/Layouts/Layout.module.scss [hunk 2]
```diff
@@ -42,15 +42,15 @@ $offset: $banner-height + map-get($spacers, 3);
...
     display: flex;
     align-items: center;
     flex-grow: 1;
   }
 
   .BreadcrumbContainer {
     align-items: center;
-    background-color: $body-bg;
+    background-color: $white;
```
Comment (eckdanny-osi) [inline libs/components/src/Layouts/Layout.module.scss]: revert plz

File: libs/core/scss/_custom.scss [hunk 3]
```diff
@@ -143,16 +146,18 @@ $component-active-color: $white;
...
 
 $font-family-sans-serif: "Source Sans Pro", sans-serif;
 
 $font-size-base: 1rem;
 
+// $font-weight-lighter:         lighter !default;
 // $font-weight-light:           300 !default;
 // $font-weight-normal:          400 !default;
-$font-weight-bold: 600;
+$font-weight-bold:            700;
```
Comment (mlipardo) [inline libs/core/scss/_custom.scss]: Just nit. Prettier didn't catch this?

**Comment by mlipardo** [comment]
The Guide site way looks different. 😮

File: libs/core/scss/_custom.scss (hunk 3 above)
Comment (eckdanny-osi) [reply libs/core/scss/_custom.scss]: prettier ignores this file (we want to maintain the format from `bootstrap/scss/_variables.scss`)

**Pull request merged**